src/audio/AudioEngine.ts

```diff
@@ -108,6 +110,7 @@ export abstract class AudioEngine {
 		if (audio) {
 			this._playAudio(audio);
 			audio.setAttribute('playing', true);
+			this.emit('playing', name, audio);
```

Comment:
Should probably move the event stuff to constants for the type, as well as building potential an interface for the data, to be consistent with my input manager changes

Reply:
I agree. Forgot about those changes in the AudioEngine. They will actually be removed since those events will be unreliable depending on how you use the AudioEngine. The HTML5 implementation hooks into the actual DOM events and propagates them using the emitter and this does follow the pattern used by the inputs package.

I'll clean this up later tonight, and do another sweep as well in case I missed anything else.